src/components/ItemSplitContainer/ItemSplitContainer.jsx

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from "react";
+import {MdOpenInNew} from "react-icons/md";
+
+import "./styles.scss";
+import ItemSplit from "../ItemSplit";
+import {Button, Modal} from "shards-react";
+
+function getSelectedItem(info, selectedItemId) {
+  return info.items.find((item) => item.itemId === selectedItemId);
+}
+
+class ItemSplitContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+      itemInformation: {},
+    };
+
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  componentDidMount() {
+    const { selectedItemId, bill } = this.props;
+    const selectedItem = bill.items.find((item) => item.id === selectedItemId);
+
+    const itemInformation = {
+      name: selectedItem.name,
+      cost: selectedItem.cost,
+      accounts: [],
+    };
+
+    bill.informationPerAccount
```

Comment:
I wonder if you could've made this as a selector?

Reply:
is this blocking

Reply:
I agree that it could be, but wondering if it's considered a blocking one since we don't test selectors and this method is tested this way for now (not that we can't test them, it's just that we don't) 

I'll leave that decision to you

Reply:
Though I think we need to have the active item as a state as well so that the selector can take the activeItem + activeBill and get that specific item's information